docs/data/pages.ts

```diff
@@ -17,9 +17,10 @@ const pages: MuiPage[] = [
       {
         pathname: '/toolpad/examples',
         children: [
-          { pathname: '/toolpad/examples/npm-stats', title: 'npm stats' },
+          { pathname: '/toolpad/examples/npm-stats', title: 'NPM stats' },
           { pathname: '/toolpad/examples/basic-crud-app', title: 'Basic CRUD application' },
           { pathname: '/toolpad/examples/qr-generator', title: 'QR Code generator' },
+          { pathname: '/toolpad/examples/more-examples', title: 'More examples' },
```

Comment:
What if we were to use this as the overview page, listed first in the side nav and listing all the examples? This would provide an introduction point, to browse from.

```suggestion
          { pathname: '/toolpad/examples/', title: 'Overview' },
```

Reply:
I agree, it would be better. On the example card, I want to show all components that each example uses, and then link this Overview page from every component [page](https://mui.com/toolpad/reference/components/button/).

Users who directly land on the Components page will get a chance to explore Toolpad and they'll also see the link to go to the Material UI component page. https://github.com/mui/mui-toolpad/pull/2451